compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -483,7 +487,9 @@ object desugar {
         val enumCompanionRef = TermRefTree()
         val enumImport =
           Import(enumCompanionRef, enumCases.flatMap(caseIds).map(ImportSelector(_)))
-        (enumImport :: enumStats, enumCases, enumCompanionRef)
+        val enumLabelDef = DesugarEnums.enumLabelMeth(EmptyTree)
+        val ordinalDef = DesugarEnums.ordinalMeth(EmptyTree).withAddedFlags(Synthetic)
```

Comment:
Maybe immediately create `ordinalDef` and `enumLabelDef` with a `Synthetic` flag? I am not sure. It does not matter much either way.

Reply:
both enumLabelMeth and ordinalMeth now add synthetic flag